src/posts/2016-08-08-hello-world.md

```diff
@@ -1,12 +1,13 @@
----
-layout: post
-title: Hello World!
-author: michael
-min_read: 1
----
-
-Vanaf vandaag zijn wij met onze blog overgegaan op Jekyll, het perfecte platform om code-style te bloggen:
-
-> Jekyll is a simple, blog aware, static site generator. It takes a template directory [...] and spits out a complete, static website suitable for serving with Apache or your favorite web server. This is also the engine behind GitHub Pages, which you can use to host your project’s page or blog right here from GitHub.
-
-[Bekijk het Jekyll project op GitHub](https://github.com/jekyll/jekyll).
+---
+title: Hello World!
+author: michael
+min_read: 1
+date: 2016-08-08
+permalink: /2016/08/08/hello-world/
+---
+
+Vanaf vandaag zijn wij met onze blog overgegaan op Jekyll, het perfecte platform om code-style te bloggen:
+
+> Jekyll is a simple, blog aware, static site generator. It takes a template directory [...] and spits out a complete, static website suitable for serving with Apache or your favorite web server. This is also the engine behind GitHub Pages, which you can use to host your project’s page or blog right here from GitHub.
+
+[Bekijk het Jekyll project op GitHub](https://github.com/jekyll/jekyll).
```

Comment:
Requires new post;
> Vanaf vandaag zijn wij met onze blog overgegaan op **Eleventy [11ty]**, het perfecte platform om code-style te bloggen:

Reply:
Hehe, when we're migrated feel free to post a new blog;ost! :-D